esx_service/cli/vmdkops_admin.py

```diff
@@ -119,24 +119,48 @@ def commands():
              defined. For example, `tenant create` has a callback, but if a user runs the program
              like: `./vmdkops_admin.py tenant` they will get the following error:
              ```
-             usage: vmdkops_admin.py tenant [-h] {rm,create,set,ls,get} ...
+             usage: vmdkops_admin.py tenant [-h] {rm,create,volume,get} ...
```

Comment:
@lipingxue  - cleaning up 'tenant->vm-group' terminology conversion is out of scope for this PR. The PR is targeting 'ls' and 'set' commands conversion to 'volume ls' and 'volume set'. 
I just fixed a couple of locations I noticed with old "tenant" in messages. for common good. If you feel it should not be there I will be happy to rolllback these, but I do not plan to put tenant-related cleanup heree